x-pack/legacy/plugins/siem/public/components/link_icon/index.tsx

```diff
@@ -52,14 +53,25 @@ export interface LinkIconProps extends LinkProps {
 }
 
 export const LinkIcon = React.memo<LinkIconProps>(
-  ({ children, color, disabled, href, iconSide = 'left', iconSize = 's', iconType, onClick }) => (
+  ({
+    children,
+    color,
+    disabled,
+    href,
+    iconSide = 'left',
+    iconSize = 's',
+    iconType,
+    onClick,
+    ariaLabel,
+  }) => (
     <Link
       className="siemLinkIcon"
       color={color}
       disabled={disabled}
       href={href}
       iconSide={iconSide}
       onClick={onClick}
+      aria-label={ariaLabel ?? children}
```

Comment:
why do we use `children` here?

Reply:
Because the `aria-label` is optional and the `children` is a `string`. So because the component is used all over the place I did not want to break anything. Do you believe we should use something else?

Reply:
Maybe we should rename the props children to something else so it is less confusing

Reply:
This prop is the React's prop `children`. If I change it I have to change the usage (`<LinkIcon>{somthing_here_as_children}</LinkIcon>`) all over the place.

Reply:
so, It will wait